libr/include/r_util/r_str.h

```diff
@@ -33,6 +33,8 @@ typedef struct r_charset_t {
 	Sdb *db_char_to_hex;
 	RCharsetRune *custom_charset;
 	size_t remaining;
```

Comment:
Whats remaining for?

Reply:
It was used for RCharset search tree before we moved to the sdb. I can remove it but if in the future we plan to make a search tree for the Sdb like I did for RCharsetRune it might be fine. I still do not know if make a serch tree for reverse sdb like I did for RCharsetRune is fine.

Reply:
Remove it. Its unised tand when using a tree it wont be needed either i think